test/Dockerfile

```diff
@@ -21,5 +21,9 @@ COPY test/test ./test/
 
 ENV PATH="/app/result/bin:$PATH"
 
+# Bats is already included in the nix environment
+# No need to install it separately as it's defined in test/packages.nix
+
 # Make scripts executable and run tests
-RUN bats test
+# Use nix-shell to ensure bats, zsh, and fish are in the PATH
+RUN nix-shell -p bats zsh fish --run "bats test"
```

Comment:
You added test/package.nix to image, so you can run just bats test now.
please revert this change